test/unit/lib/plugins/aws/package/compile/events/cognitoUserPool.test.js

```diff
@@ -339,6 +339,77 @@ describe('AwsCompileCognitoUserPoolEvents', () => {
                 Trigger: 'CustomMessage',
               },
             ],
+            ForceDeploy: undefined,
+          },
+        });
+      });
+    });
+
+    it('should create the necessary resources for the most minimal configuration with forceDeploy', () => {
+      awsCompileCognitoUserPoolEvents.serverless.service.functions = {
```

Comment:
Sorry, didn't notice that one earlier.

In new tests we always rely on `runServerless` util (we have already a tons of tests configured for it, which can self as reference).

Can you update that one, so it's also `runServerless` based?

Reply:
Sure! I had to do some reading because I'm not familiar at all with testing in Javascript (or the internals of Serverless) but I came up with this:

```
    it('should create the necessary resources for the most minimal configuration with forceDeploy', () =>
      runServerless({
        fixture: 'cognitoUserPool',
        configExt: {
          functions: {
            existingSimple: {
              events: [
                {
                  cognitoUserPool: {
                    forceDeploy: true
                  },
                },
              ],
            }
          }
        },
        command: 'package',
      }).then(({ awsNaming, cfTemplate }) => {
        const { Resources } = cfTemplate;
        let customResource = Resources[awsNaming.getCustomResourceCognitoUserPoolResourceLogicalId('existingSimple')];
        
        expect(customResource.Properties.ForceDeploy).to.not.deep.equal(undefined);
    }));
```

(only `expect()` statement instead of the previous three). Will something like this do?

Reply:
Yes, that looks very good.
Just use async/await (instead of `.then()`), and rather confirm on property as `expect(typeof ..).to.equal('string')`